### PR TITLE
[CP] Fixed lock and condition tasks and enabled tests

### PR DESCRIPTION
**Pull request by eminn**

### CODE REVIEW

File: hazelcast-client-new/src/main/java/com/hazelcast/client/proxy/ClientLockProxy.java [hunk 1]
```diff
@@ -87,7 +87,7 @@ public void lock(long leaseTime, TimeUnit timeUnit) {
     }
 
     public void forceUnlock() {
-        ClientMessage request = LockForceUnlockParameters.encode(getName());
+        ClientMessage request = LockForceUnlockParameters.encode(getName(), ThreadUtil.getThreadId());
```
Comment (sancar) [inline hazelcast-client-new/src/main/java/com/hazelcast/client/proxy/ClientLockProxy.java]: I removed this threadId intentionally before. Is it necessary to pass threadId ? If yes, can you show me where it is used.

Comment (gurbuzali) [inline hazelcast-client-new/src/main/java/com/hazelcast/client/proxy/ClientLockProxy.java]: force unlock should not need any threadId afaik

Comment (eminn) [inline hazelcast-client-new/src/main/java/com/hazelcast/client/proxy/ClientLockProxy.java]: You guys are right about that, my bad.

**Comment by sancar** [comment]
:+1:

**Pull request merged**